packages/web-components/src/spinner/spinner.options.ts

```diff
@@ -0,0 +1,48 @@
+export type Values<T> = T[keyof T];
+/**
+ * Spinner size options
+ * @public
+ */
+export const SpinnerSize = {
+  /**
+   * A Spinner's size can be small, tiny, extra-small, medium, large, extra-large, or huge
+   *
+   */
+  small: 'small',
+  tiny: 'tiny',
+  extraSmall: 'extra-small',
```

Comment:
nit: can we keep the options sorted from the smallest to the biggest?
```suggestion
  tiny: 'tiny',
  extraSmall: 'extra-small',
  small: 'small',
```